Utilities/bootstrap

```diff
@@ -651,6 +635,8 @@ def build_swiftpm_with_swiftpm(args, integrated_swift_driver):
     if integrated_swift_driver:
         swiftpm_args.append("--use-integrated-swift-driver")
 
+    swiftpm_args.append("--enable-parseable-module-interfaces")
```

Comment:
This currently results in warnings for any modules that are not built with library evolution enabled.  I’ll take a look at the logic in SwiftPM to see whether this flag should be conditional on whether the module is built with that flag.